highlighter/highlighter.go

```diff
@@ -0,0 +1,47 @@
+package highlighter
+
+import (
+	"github.com/mp-hl-2021/splinter/types"
+	"log"
+	"os/exec"
+)
+
+type Highlighter struct {
```

Comment:
Do I understand it correctly?
You are trying to use an unbuffered channel as a message queue? Please, don't do that.